### ISSUE TITLE
Dealing with tricky word transpositions

### ISSUE THREAD

**Issue by scottkleinman** [scottkleinman/aeme#48]
At the top of f. 52r (near the end of the second line) there is a really tricky word transposition. Any thoughts on how to deal with it? Here is my best effort so far:
```xml
<l xml:id="bodllaudmisc108.3l.131"><hi rend="touched-red">A</hi><sg type="ln"/>ke <choice>
    <orig>seint</orig>
    <reg>Seint</reg>
  </choice>
  <name type="person" ref="#bioMatthew0001">Matheu</name> ne fo&rrot;<sg/>ȝat nouȝt <hi
    rend="touched-red">þ</hi>at <choice>
    <orig>Maide</orig>
    <reg>maide</reg>
  </choice><pc> &punctelev; </pc>þei he <seg xml:id="transpose1a"><add place="sublinear"
  >&caret;</add><metamark function="transposition" rend="line" place="supralinear"/>
    were <add place="sublinear">&caret;</add></seg>
  <seg xml:id="transpose1b">d<metamark function="transposition" rend="line"
  place="supralinear"/>ed<add place="sublinear">&caret;</add></seg></l>
  <listTranspose>
    <transpose>
      <ptr target="#transpose1b"/>
      <ptr target="#transpose1b"/>
    </transpose>
  </listTranspose>
```

**Comment by dorothyk98** [comment]
Hmmm, let me have a look on Saturday and I will get back to you.

**Comment by skgoetz** [comment]
That makes sense to me, though I haven't had cause to encode something that way before. (More at traditional app crit entries.) Do you need to have d of "ded" between where seg starts and the metamark is placed?

**Comment by scottkleinman** [comment]
Yes, the placement of the metamark could probably use some work. In the MS, it doesn't exactly begin and end over the words that are to be transposed, but I wonder if that matters for encoding. One of the problems is that I can't imagine *how* this is supposed to be rendered. I considered enclosing the whole sequence in `<choice>` and actually transposing the words in `<reg>` (but I got too tired to do it). Otherwise we have to create an xslt routine to deal with transpositions. My students found an insane example where three lines were being transposed in two directions with metamarks in both margins. Can we really account for all these scenarios in our rendering algorithms?

**Comment by skgoetz** [comment]
That may be when one resorts to a textual note and lets the reader consult the facs image....
--
Sent via phone.
